tests/integration/tests/sample1/BUILD

```diff
@@ -0,0 +1,12 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_test")
+
+go_test(
+    name = "go_default_test",
+    srcs = ["sample1_test.go"],
```

Comment:
Can you hook up these tests in presubmit so they actually run?

Reply:
No, we can't do it right now. We use proxy binary here, and it's been built outside of proxy. We may pull it from somewhere but let's leave it for now.